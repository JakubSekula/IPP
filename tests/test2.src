.IPPCODE20    #naeo
MOVE GF@counter bool@true
CREATEFRAME
POPFRAME
CALL TruE
REtUrn
LT LF@counter bool@true nil@nil