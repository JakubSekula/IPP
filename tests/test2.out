<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode20">
 <instruction order="1" opcode="MOVE">
  <arg1 type="var">GF@counter</arg1>
  <arg2 type="bool">true</arg2>
 </instruction>
 <instruction order="2" opcode="CREATEFRAME"/>
 <instruction order="3" opcode="POPFRAME"/>
 <instruction order="4" opcode="CALL">
  <arg1 type="label">TruE</arg1>
 </instruction>
 <instruction order="5" opcode="RETURN"/>
 <instruction order="6" opcode="LT">
  <arg1 type="var">LF@counter</arg1>
  <arg2 type="bool">true</arg2>
  <arg3 type="nil">nil</arg3>
 </instruction>
</program>